lucene/core/src/java/org/apache/lucene/index/MergePolicy.java

```diff
@@ -196,6 +200,7 @@ final void setMergeThread(Thread owner) {
    *
    * @lucene.experimental */
   public static class OneMerge {
```

Comment:
Maybe we should (later, in dedicated PR) pull this out into its own java source?

Reply:
yeah I want to do that anyway